docs/src/pages/[platform]/connected-components/storage/storage-browser/react.mdx

```diff
@@ -0,0 +1,722 @@
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+  <Tabs.Panel value="Delete">
+    <Example>
+      <ComposedDeleteView />
+      <ExampleCode>
+        ```jsx file=./examples/ComposedDeleteView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+</Tabs.Container>
+
+
+### Custom UI
+
+The `createStorageBrowser` function returns a `useView()` hook that lets you use the Storage Browser's internal state and event handlers to build your own UI on top the Storage Browser functionality.
+
+The `useView()` hook takes a single argument which is the name of the view you want to get the UI state and event handlers for. The available views are `Locations`, `LocationDetails`, `Copy`, `Upload`, `Delete`, and `CreateFolder`. The return value of the hook will have all the necessary internal state and event handlers required to build that view. In fact, the Storage Browser component itself uses the `useView()` to manage its state, so you can build the UI exactly how we do!
+
+<Example>
+  <Custom />
+  <ExampleCode>
+    ```jsx file=./examples/Custom.tsx
+    ```
+  </ExampleCode>
+</Example>
+
+Below you will find examples of building custom UI for each view:
+
+<Tabs.Container defaultValue="Locations">
+  <Tabs.List>
+    <Tabs.Item value="Locations">Locations</Tabs.Item>
+    <Tabs.Item value="LocationDetail">Location Detail</Tabs.Item>
+    <Tabs.Item value="Upload">Upload</Tabs.Item>
+    <Tabs.Item value="Copy">Copy</Tabs.Item>
+    <Tabs.Item value="CreateFolder">CreateFolder</Tabs.Item>
+    <Tabs.Item value="Delete">Delete</Tabs.Item>
+  </Tabs.List>
+  <Tabs.Panel value="Locations">
+    <Example>
+      <CustomLocationsView />
+      <ExampleCode>
+        ```tsx file=./examples/CustomLocationsView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+  <Tabs.Panel value="LocationDetail">
+    <Example>
+      <CustomLocationDetailView />
+      <ExampleCode>
+        ```tsx file=./examples/CustomLocationDetailView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+  <Tabs.Panel value="Copy">
+    <Example>
+      <CustomCopyView />
+      <ExampleCode>
+        ```jsx file=./examples/CustomCopyView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+  <Tabs.Panel value="CreateFolder">
+    <Example>
+      <CustomCreateFolderView />
+      <ExampleCode>
+        ```jsx file=./examples/CustomCreateFolderView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+  <Tabs.Panel value="Upload">
+    <Example>
+      <CustomUploadView />
+      <ExampleCode>
+        ```jsx file=./examples/CustomUploadView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+  <Tabs.Panel value="Delete">
+    <Example>
+      <CustomDeleteView />
+      <ExampleCode>
+        ```jsx file=./examples/CustomDeleteView.tsx
+        ```
+      </ExampleCode>
+    </Example>
+  </Tabs.Panel>
+</Tabs.Container>
+
+
+## View reference
```

Comment:
Should these lists include descriptions for the purpose of each property of the interfaces?

Reply:
They should, I was going to do that as a fast-follow. Ideally we should have JSDoc comments on them in our source code and then generate the docs from them

Reply:
Sounds good, works for me 👍